src/cppcache/src/TcrMessage.hpp

```diff
@@ -1170,7 +1170,7 @@ class TcrMessageHelper {
     }
     if (compId != expectedPartType) {
       char exMsg[256];
-      ACE_OS::snprintf(exMsg, 255,
+      std::snprintf(exMsg, 255,
```

Comment:
This change violates the C++ formatting we adhere to. Please reformat each changed line or file after global search and replace.